CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contirbution Guidelines
+
+netlink accepts contributions via GitHub pull requests. This document outlines some of the conventions on how to contribute
+code to the project
+
+## General
+
+* One commit per PR.
+* A PR covers a single area/functionality (e.g  extension of devlink command and link command should go in separate PRs )
```

Comment:
remove the extra space before the closing parenthesis

Reply:
done.